dart/test/scope_test.dart

```diff
@@ -0,0 +1,238 @@
+import 'package:sentry/sentry.dart';
+import 'package:test/test.dart';
+
+void main() {
+  final fixture = Fixture();
+
+  test('sets $SeverityLevel', () {
```

Comment:
@rxlabz I see many people using `groups`, but most of the time a single group, does it even make sense? so I decided to not use it as the class is specific for this class and I can't group things here, maybe the breadcrumbs ring buffer could be grouped though, wdyt?

Reply:
groups give `setUp()`& `teardown()`but it's fine to not use them if not needed

Reply:
only with groups?
https://github.com/getsentry/sentry-dart/blob/main/flutter/test/sentry_flutter_test.dart
and there are no groups in there

Reply:
you right 😶 !